### ISSUE TITLE
Worse results of PyTorch reimplementation

### ISSUE THREAD

**Issue by DAA233** [JiahuiYu/generative_inpainting#202]
Hi, @JiahuiYu!

I am interested in your awesome work and trying to reimplement it with PyTorch framework. However, I got some worse results when inpainting 256x256 image patches from ImageNet validation dataset with [my PyTorch implementation](https://github.com/DAA233/generative-inpainting-pytorch). When testing the models, only 128x128 center hole mask has been used.

Here are some results for comparison. From the left to the right, they are **original image patches**, **pre-trained model results** and **reproduce results** respectively. 

![result-1](https://user-images.githubusercontent.com/15101533/50642622-b3f9ef80-0fa6-11e9-9662-76d0dadfd551.png)
![result-2](https://user-images.githubusercontent.com/15101533/50642623-b4928600-0fa6-11e9-947e-499740198636.png)
![result-3](https://user-images.githubusercontent.com/15101533/50642624-b4928600-0fa6-11e9-8457-50af20c8601c.png)
![result-4](https://user-images.githubusercontent.com/15101533/50642625-b4928600-0fa6-11e9-852a-9ef786c46310.png)
![result-5](https://user-images.githubusercontent.com/15101533/50642626-b52b1c80-0fa6-11e9-8ec7-d77bb4e81c89.png)

Like the last two images show, many inpainted results of my reproduce contain strange "color blocks", which make the results worse. Of course, there may be a lot of bugs in my codes and I am reviewing them.

I wonder if you have also met the same problem during your work. It would be helpful if you could give me some possible guesses or explanation for it. Thank you!

**Comment by JiahuiYu** [comment]
Hey thanks for your interests in our work. However, I don’t have any idea why results look like these examples.

Reproducing using PyTorch can help you understand our work better, meanwhile it requires investigation into many details. Good luck!
